### PR TITLE
Publicize

### PR DESCRIPTION
**Pull request by Viciberty**
更新了宣传页面

### CODE REVIEW

**Comment by huxuan** [comment]
Request @JeldorPKU for review to confirm the content.

**Comment by huxuan** [comment]
Another concern is that whether it is possible to publish the content via editor as we may change the content frequently.

File: application/controllers/index.php [hunk 1]
```diff
@@ -51,4 +51,12 @@ public function activity() {
         $this->load->view('activity_notification', $data);
         $this->load->view('footer');
     }
+    public function publicize() {
+        $this->load->view('header_homepage');
+        $this->load->view('add_hilight_nav5');
+        $this->load->view('publicize_2016');
+//        $this->load->view('sample');
```
Comment (JeldorPKU) [inline application/controllers/index.php]: Delete the useless comment.

**Comment by JeldorPKU** [comment]
> Another concern is that whether it is possible to publish the content via editor as we may change the content frequently.

 @huxuan As requested, we have to make this page somewhat pretty, so I think static pages are enough for this year. There are not so many pictures and the content won't change this year.

**Comment by huxuan** [comment]
If the images are not intended to be included in version control. I would suggest we create a new pull request without the images. As you may know, deleting images via commit will not delete them in the commit history, which will enlarge our git repo size easily.

I am ok with static page, just want to have a confirmation that we are on the same page.

**Comment by huxuan** [comment]
Closed as a new version pull request is sent #235

**Pull request closed without merging**